### DISCUSSION TITLE
Add custom HTML meta tag

### DISCUSSION THREAD

**Discussion by shenweiyan** [squidfunk/mkdocs-material#7431] [Q&A] [answered]
I want to add the following HTML tag validation between the `<head>` and `</head>` tags of my website. How can I achieve this?
```
<meta name="bd-site-verification" content="codeva-hAB8gPWXkn" />
```

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @shenweiyan,
I'm sure this was asked several times in some way before, but search doesn't show anything exact like your question.
If the meta tag is constant, the content doesn't change on the whole website then simply use customization with blocks:
https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks

1. Set `theme: custom_dir: overrides` in your `mkdocs.yml`
2. Create the `overrides` directory next to the `mkdocs.yml`
3. Add a `main.html` file inside that directory with the following override:
```jinja2
{% extends "base.html" %}

{% block extrahead %}
  <meta name="bd-site-verification" content="codeva-hAB8gPWXkn" />
{% endblock %}
````
The `base.html` provides the `extrahead` block and you inject your contents inside:
https://github.com/squidfunk/mkdocs-material/blob/d5363ad000d049b25ce0f66c3781040d277bfa7f/src/templates/base.html#L172-L174

**Reply by shenweiyan** [reply]
Thank you. 
I tried this in mkdocs-1.6.0 and mkdocs-material-9.5.31, but unfortunately it didn't work. I don't know what went wrong.

**Reply by kamilkrzyskow** [reply]
Here is a minimal example:
[9.5.31-overrides.zip](https://github.com/user-attachments/files/16559882/9.5.31-overrides.zip)

You can also build a project from scratch and see where the issue is and/or send the archive here.
https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#creating-a-reproduction

**Reply by shenweiyan** [reply]
Thank you very much. After troubleshooting, my problem has finally been successfully resolved.
